docs/examples/good_practices/many_tasks_per_gpu/job.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#SBATCH --gpus=1
```

Comment:
Basé sur la suggestion de https://github.com/mila-iqia/mila-docs/pull/228/files#r1372574546
```suggestion
#SBATCH --gres=gpu:rtx8000:1
```